src/text-field/README.md

```diff
@@ -2,7 +2,33 @@
 
 The `vscode-text-field` is a web component implementation of a [text field element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input/text).
 
-## Attributes
+![Text field hero](/docs/assets/images/text-field-hero.png)
+
+## Usage
+
+| ❌ Don't                                                                                                                                    | ✅ Do                                                                     |
+| ------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------- |
+| ![Text field with overflowing text](/docs/assets/images/text-field-dont-1.png)                                                              | ![Text field with a short value](/docs/assets/images/text-field-do-1.png) |
+| Don't use a text field for inputs that are greater than a single line of text. Use a [text-area](../text-area/README.md) component instead. | Use text fields for short input values.                                   |
+
+| ❌ Don't                                                                        | ✅ Do                                                                           |
+| ------------------------------------------------------------------------------- | ------------------------------------------------------------------------------- |
+| ![Text field without a formal label](/docs/assets/images/text-field-dont-2.png) | ![Text label with a descriptive label](/docs/assets/images/text-field-do-2.png) |
+| Don't use a placeholder value instead of a label unless absolutely necessary.   | Use descriptive labels to help users understand the text field's purpose.       |
```

Comment:
Again I think "text field's" should be "text fields"??

Reply:
Lol actually I feel confident that it should not be "text field's" because Grammarly marks it as an error when I write it out.